### PR TITLE
fix link for fabric8 console

### PR DESCRIPTION
**Pull request by kadel**
Fabric8 console is running only on HTTP

### CODE REVIEW

File: add-ons/fabric8/fabric8.addon [hunk 1]
```diff
@@ -34,4 +34,4 @@ echo Or you can watch in the OpenShift console via:
 echo   minishift console
 echo -
 echo Then you should be able the open the fabric8 console here:
-echo   https://fabric8-fabric8.#{routing-suffix}/
+echo   http://fabric8-fabric8.#{routing-suffix}/
```
Comment (praveenkumar) [inline add-ons/fabric8/fabric8.addon]: @kadel Does route created based on http instead of https for this addon?

Comment (kadel) [reply add-ons/fabric8/fabric8.addon]: [Fabric8 Route](https://github.com/fabric8io/fabric8-resources/blob/8e26375977614fb8a833f1514c2a442fc562c8bf/fabric8-system-openshift.yml#L2732) is HTTP only.

Comment (praveenkumar) [reply add-ons/fabric8/fabric8.addon]: Ah right,

Comment (praveenkumar) [reply add-ons/fabric8/fabric8.addon]: 👍

**Pull request merged**